Mail-Merge/Create-Envelopes-for-mailing/Console-App-.NET-Core/Create-Envelopes-for-mailing.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
Change the folder name as .NET-Standard or .NET-Framework instead of Console-App-.NET.Core or Console-App-.NET-Framework

Do the same for all samples.

Reply:
Addressed.